microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md

```diff
@@ -45,6 +48,36 @@ The top of each context menu now shows the title again:
 ![New context menu header](./devtools-128-images/header-title.png)
 
 
+<!-- ====================================================================== -->
+## Heap snapshot improvements
+
+<!-- Subtitle: Heap snapshots are generated faster, HTML elements are grouped by tag name, shallow sizes show fewer internal objects, and numeric values are always shown. -->
+
+In the **Memory** tool, there are several improvements for heap snapshots:
+
+* Generating heap snapshots is now faster.
+
+* HTML elements are grouped by tag name.<!-- todo: by element name? -->
+
+* Shallow sizes now match JavaScript semantics better, by showing fewer internal objects, unless the **Expose internals (includes additional implementation-specific details)** setting is enabled<!--todo: checkbox is selected?  where/what is this UI control?  not found after turned experiment on, and then created snapshot --> when the [Show option to expose internals in heap snapshots](../../../experimental-features/index.md#show-option-to-expose-internals-in-heap-snapshots) experiment is turned on.
+
+* The **Include numerical values in capture** setting<!-- todo: where was this setting in the UI?  was it a checkbox? --> has been removed, and heap snapshots now always show numeric values.
```

Comment:
Where was the **Include numerical values in capture** setting in the v127- UI?  was it a checkbox?

Reply:
Yes, another checkbox before taking the heap snapshot: 
![image](https://github.com/user-attachments/assets/e34c426a-232a-45b6-bf1f-da273744a0b3)

Reply:
result:
```
The **Include numerical values in capture** checkbox (shown 
before you take a heap snapshot) has been removed, and heap 
snapshots now always show numeric values.
```